### PR TITLE
Add functionality for Operators

### PR DESCRIPTION
**Pull request by mbax**

### CODE REVIEW

File: src/main/java/org/kitteh/irc/client/library/command/OperCommand.java [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    }
+
+    /**
+     * Sets the password for the command.
+     *
+     * @param password password for the user
+     * @return this command
+     */
+    @Nonnull
+    public OperCommand password(@Nonnull String password) {
```
Comment (kashike) [inline src/main/java/org/kitteh/irc/client/library/command/OperCommand.java]: Should `user` go above `password`?

File: src/main/java/org/kitteh/irc/client/library/command/OperCommand.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        this.user = Sanity.safeMessageCheck(user, "user");
+        return this;
+    }
+
+    @Override
+    public void execute() {
+        if (this.password == null) {
+            throw new IllegalStateException("Password not defined");
+        }
+        if (this.user == null) {
```
Comment (kashike) [inline src/main/java/org/kitteh/irc/client/library/command/OperCommand.java]: Should `user` go above `password`?

Comment (mbax) [inline src/main/java/org/kitteh/irc/client/library/command/OperCommand.java]: Here I can see it making sense.

File: src/main/java/org/kitteh/irc/client/library/command/OperCommand.java (hunk 1 above)
Comment (mbax) [inline src/main/java/org/kitteh/irc/client/library/command/OperCommand.java]: Do javadocs get sorted by method order? I was just being alphabetical here.

File: src/main/java/org/kitteh/irc/client/library/command/WallopsCommand.java [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+import org.kitteh.irc.client.library.Client;
+import org.kitteh.irc.client.library.util.Sanity;
+import org.kitteh.irc.client.library.util.ToStringer;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Sends a WALLOPS message to the server.
+ */
+public class WallopsCommand extends Command {
```
Comment (kashike) [inline src/main/java/org/kitteh/irc/client/library/command/WallopsCommand.java]: `newWallopsCommand` on `Client`?

File: src/main/java/org/kitteh/irc/client/library/command/OperCommand.java [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+import org.kitteh.irc.client.library.Client;
+import org.kitteh.irc.client.library.util.Sanity;
+import org.kitteh.irc.client.library.util.ToStringer;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Sends an OPER request to the server - MAKE SURE IT'S YOUR SERVER!
+ */
+public class OperCommand extends Command {
```
Comment (kashike) [inline src/main/java/org/kitteh/irc/client/library/command/OperCommand.java]: `newOperCommand` on `Client`?

File: src/main/java/org/kitteh/irc/client/library/command/OperCommand.java (hunk 1 above)
Comment (bendem) [inline src/main/java/org/kitteh/irc/client/library/command/OperCommand.java]: The summary is alphabetical, the description below is in the same order as in the file.

Comment (mbax) [inline src/main/java/org/kitteh/irc/client/library/command/OperCommand.java]: huh!

**Comment by mbax** [comment]
Re: new*Command on Client

I feel the methods already provided for convenience on Channel are useful due to the potentially frequent usage. Operator commands, particularly OPER, are much more special case. Should KICL provided a new*Command in the relevant class for all commands?

**Comment by mbax** [comment]
https://github.com/KittehOrg/KittehIRCClientLib/commit/6597f27ed280bf525313ec7455f4444f1ab8e97a

Moving discussion on convenience methods to new issue.

**Pull request closed without merging**